### PR TITLE
Switching progress bars page to rst

### PR DESCRIPTION
**Pull request by smithis7**
**Description**

Switches progress bars page of documentation from a notebook to RST.

**Motivation and context**

We didn't need TARDIS to run on that page.

**How has this been tested?**
- [ ] Testing pipeline.
- [ ] Other. 

**Examples**

https://smithis7.github.io/tardis/branch/progress_bars_doc/io/output/progress_bars.html

**Type of change**

- [ ] Bug fix. 
- [ ] New feature. 
- [ ] Breaking change. 
- [x] None of the above. 

**Checklist**

- [x] My change requires a change to the documentation.
    - [x] I have updated the documentation accordingly.
    - [x] (optional) I have built the documentation on my fork following [the instructions](https://tardis-sn.github.io/tardis/development/documentation_preview.html).
- [x] I have assigned and requested two reviewers for this pull request.

### CODE REVIEW

File: docs/io/output/progress_bars.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+*******************************
+Progrss Bars for Simulation Run
```
Comment (wkerzendorf) [inline docs/io/output/progress_bars.rst:2]:
```suggestion
Progress Bars for Simulation Run
```

File: docs/io/output/progress_bars.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+*******************************
+
+TARDIS displays progress bars by default to track the simulation's progress.
+
+.. image::
+    ../images/progress_bars_demo.gif
+    :width: 500
+    
+Progress bars can be turned off via the keyword argument ``show_progrss_bars=False`` in the ``run_tardis()`` function (see :doc:`../../quickstart`) or in the ``Simulation.from_config()`` class method (see :doc:`../../physics/setup/index`).
```
Comment (wkerzendorf) [inline docs/io/output/progress_bars.rst:11]:
```suggestion
Progress bars can be turned off via the keyword argument ``show_progress_bars=False`` in the ``run_tardis()`` function (see :doc:`../../quickstart`) or in the ``Simulation.from_config()`` class method (see :doc:`../../physics/setup/index`).
```

File: docs/io/output/progress_bars.rst [hunk 3]
```diff
@@ -0,0 +1,11 @@
+********************************
+Progress Bars for Simulation Run
+********************************
+
+TARDIS displays progress bars by default to track the simulation's progress.
+
+.. image::
+    ../images/progress_bars_demo.gif
+    :width: 600
+
```
Comment (atharva-2001) [inline docs/io/output/progress_bars.rst:10]: Thank you for making this change! Could this be just a little bit larger? Just enough to make words clear in the default view?

**Review by andrewfullard — APPROVED** [review]

File: docs/io/output/progress_bars.rst [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+Progress Bars for Simulation Run
+********************************
+
+TARDIS displays progress bars by default to track the simulation's progress.
+
+.. image::
+    ../images/progress_bars_demo.gif
+    :width: 750
+    
+Progress bars can be turned off via the keyword argument ``show_progress_bars=False`` in the ``run_tardis()`` function (see :doc:`../../quickstart`) or in the ``Simulation.from_config()`` class method (see :doc:`../../physics/setup/index`).
```
Comment (atharva-2001) [inline docs/io/output/progress_bars.rst:11]: Just one small thing, when https://github.com/tardis-sn/tardis/pull/1823 is merged, this and other text would have to be changed accordingly
```suggestion
Progress bars can be turned on via the keyword argument ``show_progress_bars=True`` in the ``run_tardis()`` function (see :doc:`../../quickstart`) or in the ``Simulation.from_config()`` class method (see :doc:`../../physics/setup/index`).
```

**Pull request merged**